test/models/project_media_test.rb

```diff
@@ -465,4 +465,17 @@ def perform(id)
       create_project_media media: nil, set_original_claim: video_url
     end
   end
+
+  test "should include feed_name in version_metadata when imported_from_feed_id changes" do
+    feed = create_feed name: "Feed test"
+    pm = create_project_media imported_from_feed_id: feed.id
```

Comment:
we can run the create item inside `with_versioning do` to  verify logs 
i.e 
    with_versioning do
      u = create_user
      t = create_team
      create_team_user team: t, user: u, role: 'admin'
      feed = create_feed name: "Feed test"
      pm = nil
      with_current_user_and_team(u, t) do
        assert_difference 'PaperTrail::Version.count', 4 do
          pm = create_project_media media: m, imported_from_feed_id: feed.id
        end
      end
      versions = Version.where(item_type: 'ProjectMedia', item_id: pm.id)
      # Verify meta field 
    end